integration_tests/array_expr_01.py

```diff
@@ -36,6 +36,6 @@ def array_expr_01():
     g = reshape(e + f, shape1d)
 
     for i in range(dim1d):
-        assert abs(g[i] - 2*(i + 1)) <= eps
+        assert f64(abs(g[i] - f64(2*(i + 1)))) <= eps
```

Comment:
I think the outer `f64` is not necessary? I think `abs` of f64 should return an f64.

Reply:
Removed `f64`.